packages/luxon/tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "dist",
-    "rootDir": "src"
+    "rootDir": "src",
+    "module": "esnext",
+    "outDir": "dist/esm",
+    "target": "esnext"
```

Comment:
Use `moduleResolution: node16` or later?

Reply:
Couldn't make it work. For proper ES imports in generated files, ES<something> module seems to be required.